src/components/selectPoolModal/SelectPoolModalContent.tsx

```diff
@@ -9,6 +9,9 @@ interface Props {
 }
 export const SelectPoolModalContent = ({ pools, onSelect }: Props) => {
   const [search, setSearch] = useState('');
+  const filteredPools = pools.filter((pool) =>
+    pool.reserves[0].symbol.toLowerCase().includes(search.toLowerCase())
```

Comment:
is `pool.reserves.length` always > 0?

Reply:
yeah, each pool must have 2 reseves.

pool.reserves[0] => tkn reserve
pool.reserves[1] => bnt reserve